test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1

```diff
@@ -10,8 +10,18 @@ Describe "Set/New/Remove-Service cmdlet tests" -Tags "Feature", "RequireAdminOnW
         }
         if ($IsWindows) {
             $userName = "testuserservices"
-            $Password = ([char[]]([char]33..[char]95) + ([char[]]([char]97..[char]126)) + 0..9 | Sort-Object {Get-Random})[0..12] -join ''
-            $testPass = (New-Object -TypeName Net.NetworkCredential("", $Password)).SecurePassword
+            $numbers = "0123456789"
+            $lowercase = "abcdefghijklmnopqrstuvwxyz"
+            $uppercase = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+            $symbols = "!`"#$%&'()*+,-./:;<=>?[\]^_``"
```

Comment:
Are all of these symbols allowed?

Reply:
Found Msft doc page with symbols, so I'll use that to be sure.